scarb/src/lib.rs

```diff
@@ -42,3 +42,4 @@ pub const TEST_ASSERTS_PLUGIN_NAME: &str = "assert_macros";
 pub const CAIRO_RUN_PLUGIN_NAME: &str = "cairo_run";
 pub const CARGO_MANIFEST_FILE_NAME: &str = "Cargo.toml";
 pub const CARGO_LOCK_FILE_NAME: &str = "Cargo.lock";
+pub const EXECUTABLE_PLUGIN_NAME: &str = "executable";
```

Comment:
Would you mind renaming the plugin to `cairo_execute`? I feel like it would be more idiomatic to our plugin names.

Reply:
not at all. just to be clear, you mean the value not the const
```suggestion
pub const EXECUTABLE_PLUGIN_NAME: &str = "cairo_execute";
```
??

Reply:
Yes, here and everywhere where it is needed (i.e. in scarblib templates).